source/controlTypes/state.py

```diff
@@ -96,6 +96,7 @@ def negativeDisplayString(self) -> str:
 	# indeterminate progress bar, aka busy indicator. No specific state label.
 	# when combined with role of 'progress bar', role is mutated to 'busy indicator'
 	INDETERMINATE = setBit(44)
+	HALFPRESSED = setBit(45)

```

Comment:
I think this name would read better if it had an underscore breaking it up

```suggestion
	HALF_PRESSED = setBit(45)
```